doc/classes/TextureButton.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		[TextureButton] has the same functionality as [Button], except it uses sprites instead of Godot's [Theme] resource. It is faster to create, but it doesn't support localization like more complex [Control]s.
-		The "normal" state must contain a texture ([member texture_normal]); other textures are optional.
+		While not required, setting a texture for the "normal" state ([member texture_normal]) is recommended. Otherwise, the [TextureButton] will still receive input events and be clickable, but the user will not be able to see it.
 		See also [BaseButton] which contains common properties and methods associated with this node.
```

Comment:
"Is recommended" implies it's not absolutely mandatory on its own.
Also moving the note at the bottom.
```suggestion
		See also [BaseButton] which contains common properties and methods associated with this node.
		[b]Note:[/b] Setting a texture for the "normal" state ([member texture_normal]) is recommended. Otherwise, the [TextureButton] will still receive input events and be clickable, but the user will not be able to see it.
```